Cargo.toml

```diff
@@ -58,8 +58,8 @@ debug-assertions = false
 opt-level = 3
 strip = true
 codegen-units = 1
-# lto = true
-panic = "abort"
+lto = true
```

Comment:
Just testing some stuff, will revert this line. Please do not merge until I revert this.